test-apps/ui-test-app/src/frontend/appui/widgets/NavigationTreeWidget.tsx

```diff
@@ -99,14 +100,19 @@ const NavigationTree: React.FC<NavigationTreeProps> = (props: NavigationTreeProp
   });
   const modelSource = nodeLoader.modelSource;
   const eventHandler = useUnifiedSelectionTreeEventHandler({ nodeLoader, collapsedChildrenDisposalEnabled: true });
-  const visibleNodes = useVisibleTreeNodes(modelSource);
+  const treeModel = useTreeModel(modelSource);
+  const { width, height, ref } = useResizeDetector();
   return (
-    <ControlledTree
-      visibleNodes={visibleNodes}
-      nodeLoader={nodeLoader}
-      selectionMode={SelectionMode.Single}
-      treeEvents={eventHandler}
-    />
+    <div ref={ref} style={{ width: "100%", height: "100%" }}>

```

Comment:
While `VisibilityTreeWidget` has lost `width` and `height` attributes, they remain here. Is this intentional?

Reply:
Yes. `VisibilityTreeWidget` gets its style from CSS through a class. This one doesn't.